src/Numerics/Mesh/Topologies.jl

```diff
@@ -828,14 +827,24 @@ function CubedShellTopology(
         end
     end
 
-    topology = BrickMesh.connectmesh(
-        mpicomm,
-        topology[1],
-        elemtocoord,
-        topology[3],
-        topology[4];
-        dim = 2,
-    )
+    topology =
+        connectivity == :face ?
+        BrickMesh.connectmesh(
+            mpicomm,
+            topology[1],
+            elemtocoord,
+            topology[3],
+            topology[4],
+            dim = 2,
+        ) :
+        BrickMesh.connectmeshfull(
+            mpicomm,
+            topology[1],
+            elemtocoord,
+            topology[3],
+            topology[4],
+            dim = 2,
+        )
```

Comment:
```suggestion
    connect = connectivity == :face ? BrickMesh.connectmesh : BrickMesh.connectmeshfull
    topology = connect(
        mpicomm,
        topology[1],
        elemtocoord,
        topology[3],
        topology[4],
        dim = 2,
    )
```

Reply:
Yes. In general, vertex connectivity increases the number of neighbors.

Reply:
@kpamnany yes, it will increase communication. We will probably want to add CG and DG communication patterns, can discuss on Wednesday.